README.md

```diff
@@ -29,44 +33,40 @@ scheduling appointments as easy as possible.
 
 We :heart: contributions.
 
-Have you had a good experience with this project? Why not share some
```

Comment:
I ran prettier on everything. That's why the changes. please ignore whitespace for a better experience.